packages/web-pkg/src/composables/actions/files/useFileActionsShowShares.ts

```diff
@@ -5,18 +5,26 @@ import { SideBarEventTopics } from '../../sideBar'
 import { computed } from 'vue'
 import { useGettext } from 'vue3-gettext'
 import { useRouter } from '../../router'
-import { FileAction, FileActionOptions } from '../types'
+import { FileAction } from '../types'
+import { FileActionOptionsWithEvent } from './useFileActions'
 import { useCanShare } from '../../shares'
 import { useResourcesStore } from '../../piniaStores'
+import { useInterceptModifierClick } from '../../keyboardActions'
 
 export const useFileActionsShowShares = () => {
   const router = useRouter()
   const { $gettext } = useGettext()
   const { canShare } = useCanShare()
   const resourcesStore = useResourcesStore()
+  const { interceptModifierClick } = useInterceptModifierClick()
+
+  const handler = ({ resources, event }: FileActionOptionsWithEvent & { event?: MouseEvent }) => {
```

Comment:
This should be sufficient, `FileActionOptionsWithEvent` already includes `event`.

```suggestion
  const handler = ({ resources, event }: FileActionOptionsWithEvent) => {
```